test/integration/patients/patient/patient-flow.js

```diff
@@ -814,6 +817,8 @@ context('patient flow page', function() {
         return fx;
       })
       .routeActionActivity()
+      .routePatientField()
+      .routeActionComments()
```

Comment:
These are unrelated to the topic of this PR, but were causing some 500 errors in the tests. Noticed a few more that I added throughout this PR.